Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -489,18 +515,35 @@ class DropDownMenu
     }
   }
 
-  private static ApplyPosition (popUpDiv: HTMLDivElement, clickEvent: Nullable<MouseEvent>, referenceElement: HTMLElement): void
+  private static ApplyPosition (popUpDiv: HTMLDivElement, clickEvent: Nullable<MouseEvent>, referenceElement: HTMLElement, isPeriodic: boolean): void
   {
     var space_top = Math.round (LayoutUtility.GetOffset (referenceElement).top - window.pageYOffset);
     var space_bottom = Math.round (document.documentElement.clientHeight - LayoutUtility.GetOffset (referenceElement).top - LayoutUtility.GetHeight (referenceElement) + window.pageYOffset);
     var space_left = LayoutUtility.GetOffset (referenceElement).left;
     var space_right = document.documentElement.clientWidth - LayoutUtility.GetOffset (referenceElement).left - LayoutUtility.GetWidth (referenceElement);
 
+    // Periodic update: only update our position if it has change to allow the usage of the scrollbars
+    // Other update: it might take a periodic call to get the correct position so we force it by deleting the last position information
+    if (isPeriodic) {
```

Comment:
consistency: braces in new lines

Reply:
Might be a good idea to use ESLint, what do you think?

Reply:
@a-ctor is that workable with our script files? After all, the files are much less conssitent (for various reasons) than in c#.

Reply:
yes, but in particular the braces: we have different styles in different files and different styles within some of the files. So, I'm a bit worried that enforcing brace-layout and whitespaces will mean we need to do a ton manual cleanup? Am not against enforcing these things, am just saying, it may be more involved than it was in c#

Reply:
ESLint can be used to automatically fix found issues too, so there would not be any manual cleanup needed - just a review I guess.